src/people/widgetViews/postBounty/PostModal.tsx

```diff
@@ -23,6 +23,7 @@ export const PostModal: FC<PostModalProps> = observer(
     const isMobile = useIsMobile();
     const [focusIndex, setFocusIndex] = useState(-1);
     const history = useHistory();
+    const [recallBouties] = useState(false);
```

Comment:
I'm confused why we are adding this state value if it is always going to be `false`

Reply:
changed the approach